docs/admin/projects.rst

```diff
@@ -384,36 +379,36 @@ Check resource limits
 +++++++++++++++++++++
 
 If you are importing huge translations or repositories, you might be hit by
-resource limits of your server.
+resource limitations of your server.
 
-* check amount of free memory, having translation files by system will greatly improve performance
-* disk operations might be bottleneck if there is lot of strings to process - the disk is pushed by both Weblate and database
-* additional CPU cores might help to improve performance of background tasks (see :ref:`celery`)
+* Check the amount of free memory, having translation files by the system memory will greatly improve performance.
```

Comment:
Is this correct now?

Reply:
the original sentence probably should be "having translation files cached by system"

Reply:
I've clarified this in 9916ca79f1.